src/public/less/pages/package.less

```diff
@@ -7,5 +7,18 @@
 
 	.c-slider {
 		overflow: hidden;
+		.c-slide {
```

Comment:
Change this to `.c-package` (and move the code to `less/components/c-package` to fix the issue @jimaek mentioned.

Reply:
we are thinking of trying other method. thnx